stake-pool/program/src/lib.rs

```diff
@@ -25,9 +25,6 @@ const AUTHORITY_DEPOSIT: &[u8] = b"deposit";
 /// Seed for withdraw authority seed
 const AUTHORITY_WITHDRAW: &[u8] = b"withdraw";
 
-/// Seed for transient stake account
-const TRANSIENT_STAKE_SEED: &[u8] = b"transient";
```

Comment:
I'm not sure if removing this seed and relying solely on the u64 seed is an altogether good idea.

In general, I think it's a good practice to keep namespaces separate and identifiable, as I'd hope we'd do for the transient PDAs.

Reply:
That's fair, I'll add it back in